log.c

```diff
@@ -44,7 +44,9 @@ static void fprint_timestamp(
 
 static void stderr_msg(const char *file, int line, const char *func, int priority, const char *message, const char *appendix)
 {
+#ifndef ANDROID
```

Comment:
For what reason this change is required? If you do not want to send output to stderr, why not make changes to config file?